File: Module.Frontend.TPM/Client/model/tpm/previousdayincremental/PreviousDayIncremental.js
Ext.define('App.model.tpm.previousdayincremental.PreviousDayIncremental', {
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    breezeEntityType: 'PreviousDayIncremental',
    fields: [
        { name: 'Id', hidden: true },

        { name: 'PromoId', hidden: true },
        { name: 'ProductId', hidden: true }, 
        { name: 'Week', type: 'string', hidden: false, isDefault: true, useNull: true },    
        { name: 'DemandCode', type: 'string', hidden: false, isDefault: true, useNull: true },    
        { name: 'DMDGroup', type: 'string', hidden: true, isDefault: false, useNull: true },
        { name: 'IncrementalQty', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'LastChangeDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        {
            name: 'ZREP', type: 'string', mapping: 'Product.ZREP', breezeEntityType: 'Product', hidden: false, isDefault: true, defaultFilterConfig: { valueField: 'ZREP' }
        },
        {
            name: 'Number', type: 'int', mapping: 'Promo.Number', breezeEntityType: 'Promo', hidden: false, isDefault: true, defaultFilterConfig: { valueField: 'Number' },
        },
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'PreviousDayIncrementals',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/TPM/PreviousDayIncremental.cs
using Core.Data;
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Module.Persist.TPM.Model.TPM
{
    public class PreviousDayIncremental : IEntity<Guid>
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public System.Guid Id { get; set; }

        public string Week { get; set; }
        public string DemandCode { get; set; }
        public string DMDGroup { get; set; }
        public double? IncrementalQty { get; set; }
        public DateTimeOffset? LastChangeDate { get; set; }

        public Guid? PromoId { get; set; }
        public virtual Promo Promo { get; set; }
        public Guid? ProductId { get; set; }
        public virtual Product Product { get; set; }

    }
}
